qa/suites/rgw/notifications/tasks/http/supported-distros/ubuntu_latest.yaml

```diff
@@ -0,0 +1,3 @@
+os_type: ubuntu
+os_version: "22.04"
+ktype: distro
```

Comment:
these distro files should always be symlinks, please don't create copies

`tasks/http/supported-distros` should be a symlink to `.qa/distros/supported-random-distro$`